source/FreeRTOS_Routing.c

```diff
@@ -442,22 +442,7 @@ struct xIPv6_Couple
  */
         NetworkEndPoint_t * FreeRTOS_FindEndPointOnIP_IPv6( const IPv6_Address_t * pxIPAddress )
```

Comment:
Are we mixing FreeRTOS_FindEndPointOnIP & FreeRTOS_InterfaceEndPointOnNetMask?, Shoudn't it be FreeRTOS_FindEndPointOnNetMask_IPv6

Reply:
The behaviour of `FreeRTOS_FindEndPointOnIP_IPv6` is exactly similar when `FreeRTOS_InterfaceEndPointOnNetMask` is called with interface as NULL.

Reply:
For V4 there is a difference, but in IpV6 the implementation of both is exactly same.